### ISSUE TITLE
MQTT manual add in configuration.yaml

### ISSUE THREAD

**Issue by Spenser715** [LAB02-Research/HASS.Agent#285]
I am trying to add the service to my HA but the local API isn't working for some unknown reason and my MQTT auto-discover has never worked so I've had to manually input all my MQTT devices. Can you provide information on how to manually add HASS.Agent into MQTT via configuration.yaml.

**Comment by ruifung** [comment]
Wouldn't it be... easier to fix the auto-discovery not working?

I can think of a few reasons why it wouldn't work...
1. You're using ACLs on your mqtt broker, and thats blocking either subscribing or publishing to the discovery topic.
2. You have mismatching discovery topic prefixes configured between home assistant and hass agent.

But if you're really intending on configuring it manually, easiest way is just to look at the mqtt discovery payloads sent through the broker (via something like MQTT Explorer) since that would basically contain more or less the same information used to manually configure them via configuration yaml files.

If you can't see any discovery messages via MQTT-Explorer, well, that would explain why your discovery isn't working then.

**Comment by bcutter** [comment]
"Fixed" this @Spenser715 ?

**Comment by Spenser715** [comment]
Yes i did a bunch of uninstall reinstalls and restarted everything multiple times and it eventually worked with auto discover.

**Comment by bcutter** [comment]
Same for me. After the 7th restart auto discovery worked once again and HASS.Agent could set up a new instance.
